src/main/java/com/nike/cerberus/jobs/HystrixMetricsProcessingJob.java

```diff
@@ -0,0 +1,160 @@
+/*
+ * Copyright (c) 2017 Nike, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.nike.cerberus.jobs;
+
+import com.google.inject.Inject;
+import com.google.inject.Singleton;
+import com.netflix.hystrix.HystrixCircuitBreaker;
+import com.netflix.hystrix.HystrixCommandMetrics;
+import com.netflix.hystrix.HystrixThreadPoolMetrics;
+import com.nike.riposte.metrics.codahale.SignalFxAwareCodahaleMetricsCollector;
+import com.signalfx.codahale.metrics.SettableLongGauge;
+import org.knowm.sundial.Job;
+import org.knowm.sundial.exceptions.JobInterruptException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.Optional;
+
+/**
+ * Periodically print Hystrix metrics to the log.
+ */
+@Singleton
+public class HystrixMetricsProcessingJob extends Job {
+
+    private static final Logger log = LoggerFactory.getLogger(HystrixMetricsProcessingJob.class);
+
+    private final SignalFxAwareCodahaleMetricsCollector metricsCollector;
+
+    @Inject
+    public HystrixMetricsProcessingJob(SignalFxAwareCodahaleMetricsCollector metricsCollector) {
```

Comment:
You want to use the generic `CodahaleMetricsCollector` class here in case SignalFx metrics are disabled.